gradlew

```diff
@@ -205,6 +205,13 @@ set -- \
         org.gradle.wrapper.GradleWrapperMain \
         "$@"
 
+
+# Stop when "xargs" is not available.
+if ! command -v xargs >/dev/null 2>&1
+then
+    die "xargs is not available"
+fi
+
 # Use "xargs" to parse quoted args.
```

Comment:
Are these changes down to the gradle update above? Just ensuring these are not local changes.